### ISSUE TITLE
Для компоненту реєстрації на тур покращити досвід користувача (UX)

### ISSUE THREAD

**Issue by msenyk** [msenyk/sf-practicum-2022#18]
Передбачити відображення:
- доступна кількості місць на тур. Поле, значення якого змінюється, коли користувач обирає інший доступний тур
- повідомлення про відстутність вільних місць, якщо взаалі немає жодного туру
- повідомлення валідації, коли на заданий тур користувач намагається забронювати більше квитків, ні є в наявності.

**Comment by thxnk** [comment]
Нова назва компоненту: anonimusRegistrationFormOnTour
Логіка виводу повідомлення валідації трохи крива. Думав вивести просто ToastEvent, коли натискають кнопку, якщо неправильна кількість квитків, але це було б якось не правильно.

**Comment by thxnk** [comment]
Усе зроблено.

**Comment by msenyk** [comment]
@thxnk Компонент виглядає чудово. Єдине що варто виправити це відображення у списку доступних турів. Зараз там день + назва туру. А має бути день + назва типу екскурсії. Тобто замість "Feb 02 2022-02-02 Test2" має бути "Feb 02 - Test2"

**Comment by thxnk** [comment]
2022-02-02 Test2 - це і є назва туру. Це треба виправити на стороні бази даних.

**Comment by msenyk** [comment]
@thxnk Так, це назва туру. А треба взяти назву не тура, а назву типу екскурсії.
